### PR TITLE
Add a services section to metaphysics

### PR DESCRIPTION
**Pull request by orta**
I was thinking something like this ( the code can definitely be refactored ( the value will always be a string, so can probably generate the schema obj, resolver and type from one object) and it has no code, but it's enough to show what I mean

screen shot 2017-09-18 at 11 05 31 am

### CODE REVIEW

**Comment by mzikherman** [comment]
In Slack, we were discussing an immediate use case of this (Emission/Consignments making Gemini API calls, and thus wanting to consolidate some env/config), might be better handled by just actually implementing the required image processing Gemini calls directly within Metaphysics. That sort of sidesteps the general question posed by this PR, but was worth noting (in general: how/which services should be wrapped by Metaphysics altogether, vs exposing information for clients to operate directly).

**Comment by orta** [comment]
Additional Slack context too:
* 1st mention: https://artsy.slack.com/archives/C1HH3KNJG/p1505234905000374
*  larger discussion: https://artsy.slack.com/archives/C1HH3KNJG/p1505751412000459

File: schema/services/gemini.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+import { GraphQLString, GraphQLObjectType } from "graphql"
+
+const GeminiSchema = new GraphQLObjectType({
+  name: "Gemini",
+  fields: () => ({
+    api_base: {
+      type: GraphQLString,
+    },
+    cloudfront_url: {
+      type: GraphQLString,
```
Comment (alloy) [inline schema/services/gemini.js]: I assume these should be non-null?

**Comment by alloy** [comment]
Is this still needed or superseded by #759?

**Comment by orta** [comment]
They're superseded, the thing is, this is still a good idea though, but I don't wanna ship code that doesn't need to be in production for no reason

**Comment by alloy** [comment]
That makes sense. So how do you wanna proceed?

**Comment by orta** [comment]
Let's close it, the implementation is trivial to recreate in the future if/when this is needed.

**Pull request closed without merging**